README.Rmd

```diff
@@ -122,6 +123,30 @@ imeta(
 # check if file is stored with associated metadata
 ils(metadata = TRUE)
 ```
+#### Reading metadata
+Assume you have a well-annotated object and you would like to use some of the metadata in your analysis. How can you read the metadata of "foo" into an R table?
+In the code below we extract the metadata table which the `ils` command provides us with and assign it to a new variable we can further work with:
+
+```r
+foo_meta <- ils(path="foo", metadata = TRUE)
```

Comment:
Bit of a stickler here, but could you adhere to the tidyverse styleguide: https://style.tidyverse.org/syntax.html

```
foo <- ils(path = "foo", metadata = TRUE)
```

Reply:
I am not sure what you are hinting at. I used `foo_meta` since I am not pulling `foo` from iRODS but its metadata,. hence `foo_meta`. 
I think for the table I can rename the variable I store the table in into `foo_metatable`. Is that what you meant?

Reply:
No, I mean spaces in between operators and variables. If you use RStudio, you can just use `CTRL + SHIFT +A` that will do automatic syntax formatting.

Reply:
Ah, I see. I hope now all is formatted correctly.